lang/attribute/account/src/lib.rs

```diff
@@ -79,6 +80,8 @@ pub fn account(
             .collect();
         if ns == "zero_copy" {
             is_zero_copy = true;
+        } else if ns == "packed" {
```

Comment:
The `account` attribute only takes two args but it can't take a namespace so I think this is ok?